pkg/signup/service/signup_service_test.go

```diff
@@ -1608,6 +1608,81 @@ func (s *TestSignupServiceSuite) TestGetSignupReadyConditionStatus() {
 	}
 }
 
+func (s *TestSignupServiceSuite) TestGetSignupBannedUserEmail() {
+	// given
+	s.ServiceConfiguration(configuration.Namespace(), true, "", 5)
+
+	us := s.newBannedUserSignup()
+	us.Name = service.EncodeUserIdentifier(us.Spec.IdentityClaims.PreferredUsername)
+	err := s.FakeUserSignupClient.Tracker.Add(us)
+	require.NoError(s.T(), err)
+
+	c, _ := gin.CreateTestContext(httptest.NewRecorder())
+
+	toolchainStatus := s.newToolchainStatus(".apps.")
+	err = s.FakeToolchainStatusClient.Tracker.Add(toolchainStatus)
+	require.NoError(s.T(), err)
+
+	bannedUserID, err := uuid.NewV4()
+	require.NoError(s.T(), err)
+
+	err = s.FakeBannedUserClient.Tracker.Add(&toolchainv1alpha1.BannedUser{
+		TypeMeta: v1.TypeMeta{},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      bannedUserID.String(),
+			Namespace: configuration.Namespace(),
+			Labels: map[string]string{
+				toolchainv1alpha1.BannedUserEmailHashLabelKey: "a7b1b413c1cbddbcd19a51222ef8e20a",
+			},
+		},
+		Spec: toolchainv1alpha1.BannedUserSpec{
+			Email: "jsmith@gmail.com",
+		},
+	})
+	require.NoError(s.T(), err)
+
+	rr := httptest.NewRecorder()
+	ctx, _ := gin.CreateTestContext(rr)
```

Comment:
this looks similar to what's in line 1620

Reply:
Good catch 🤦‍♂️ 

Cleaned in https://github.com/codeready-toolchain/registration-service/pull/404/commits/72752b29c173cfe6137f0ac2a6b56c58a02707c6